jwst/ami/nrm_core.py

```diff
@@ -63,13 +59,22 @@ def __init__(self, instrument_data, **kwargs):
             self.npix = kwargs["npix"]
         else:
             self.npix = 'default'
+        # Default: unweighted fit
+        self.weighted = False    
+        if "weighted" in kwargs:
```

Comment:
I know this wasn't introduced by this PR, but this seems to me a strange way to use **kwargs.  The long chain of if...else statements could be eliminated by listing each of the three keyword arguments as optional arguments to the function with default values.

Reply:
This was refactored in https://github.com/spacetelescope/jwst/pull/7862/commits/f982ebfc446c7e7552124f1dd12e5debcb1c2e94